.github/workflows/docker-build-push-github.yaml

```diff
@@ -36,13 +34,14 @@ on:
 
 jobs:
   docker-build-push-github:
+    name: github
     runs-on: ${{fromJSON(inputs.runner)}}
     permissions:
       contents: read
       id-token: write
       packages: write
     env:
-      image-path: ${{ inputs.registry }}/${{ github.repository }}/${{ inputs.image }}
+      image-path: ${{ inputs.registry }}/${{ github.repository }}/${{ inputs.subpath }}
```

Comment:
@jazzlyn  it's not missing filename here? In case of previous Implementation it set image now if su path is not defined how will result look like?

Reply:
subpath is a required input and needs at least the image name